auslib/db.py

```diff
@@ -1004,7 +1004,7 @@ def __init__(self, db, metadata, dialect):
                            Column('rule_id', Integer, primary_key=True, autoincrement=True),
                            Column('alias', String(50), unique=True),
                            Column('priority', Integer),
-                           Column('mapping', String(100)),
+                           Column('mapping', String(100), nullable=True),
```

Comment:
Actually `ForeignKey('releases.name', ondelete="SET NULL"), nullable=False `
was supposed to return an error when Parent is deleted. As it did not work, will removing ForeignKey I must have set nullable to True. It isn't  needed I will remove it.